WordPress/Classes/Utility/ZendeskUtils.swift

```diff
@@ -49,34 +50,34 @@ extension NSNotification.Name {
     private static var zdClientId: String?
     private static var presentInController: UIViewController?
 
-    private static let sourcePlatform = "mobile_-_ios"
     private static var appVersion: String {
         return Bundle.main.shortVersionString() ?? Constants.unknownValue
     }
 
     private static var appLanguage: String {
-        guard let lang = Locale.current.languageCode else {
-            return Constants.unknownValue
-        }
-        return lang
+        return Locale.preferredLanguages[0]
```

Comment:
I see `preferredLanguages` returns a non-Optional array. In my testing, this was never empty. Is there a sensible default we could use for that unlikely scenario?

Reply:
Well, I chose this method of getting the language specifically because it does always returns a value, so a default value isn't necessary.

Reply:
Fair enough. I'll trust your discretion.